<?xml version="1.0" standalone="yes"?>
<pysystest type="auto" state="runnable">
    
  <description> 
    <title>pysys.py - run - syntax error handling</title>    
    <purpose><![CDATA[
Checks that we get a full error including a traceback if the project file cannot be loaded 
due to a python coding problem. 
]]>
    </purpose>
  </description>

  <classification>
    <groups>
      <group>process</group>
    </groups>
  </classification>

  <data>
    <class name="PySysTest" module="run"/>
  </data>
  
  <traceability>
    <requirements>
      <requirement id=""/>     
    </requirements>
  </traceability>
</pysystest>
